Model/Importer.php

```diff
@@ -43,6 +43,7 @@ public function __construct(
             'ignore_duplicates'                => $this->config->getIgnoreDuplicates(),
             'validation_strategy'              => $this->config->getValidationStrategy(),
             'allowed_error_count'              => $this->config->getAllowedErrorCount(),
+            'category_path_seperator'          => $this->config->getCategoryPathSeperator(),
```

Comment:
Are you sure this changes something? Because from https://github.com/firegento/FireGento_FastSimpleImport2/blob/develop/Model/Import/Category.php, it looks like `category_path_seperator` is read from the system configuration anyway. Are these `settings` actually used somewhere?

Reply:
Checked again. As far as I see, most of the entries here are not used at all, indeed.
But did not want to change more, than I actually intended.
So I changed the getter now properly, so we don't read it directly from the config all the time, but have one spot.

Reply:
@sprankhub , I checked again. This is definitely not used, you're right. So I removed it again. But in general I would say, the config getter would be good anyway to have it all the same way.